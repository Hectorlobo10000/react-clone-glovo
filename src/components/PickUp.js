/** @jsx jsx */
import { jsx } from "@emotion/core";
import { Input } from "../components/ui";

function PickUp() {
  return (
    <div>
      <Input
        css={{
          margin: "2rem 0"
        }}
        aria-label="Restaurant Adress"
        type="text"
        value="Jorge Chavez 184 - Miraflores"
        disabled
      />
    </div>
  );
}

export default PickUp;
