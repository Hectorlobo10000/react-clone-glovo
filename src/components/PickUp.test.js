import React from "react";
import { render } from "@testing-library/react";
import PickUp from "../views/PickUp";
test("PickUp Component", () => {
  const { asFragment } = render(<PickUp />);
  expect(asFragment()).toMatchSnapshot();
});
