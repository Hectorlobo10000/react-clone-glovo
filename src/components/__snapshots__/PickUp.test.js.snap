// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PickUp Component 1`] = `
<DocumentFragment>
  .emotion-0 {
  background: white;
  border: 1px solid #eaeaea;
  border-radius: .25rem;
  text-align: center;
  box-sizing: border-box;
  display: block;
  font-size: 1.5rem;
  padding: .5rem;
  width: 100%;
  margin: 2rem 0;
}

.emotion-0:focus {
  outline: none;
}

<div>
    <input
      aria-label="Restaurant Adress"
      class="emotion-0"
      disabled=""
      type="text"
      value="Jorge Chavez 184 - Miraflores"
    />
  </div>
</DocumentFragment>
`;
